packages/theme-chalk/src/input.scss

```diff
@@ -208,7 +208,7 @@
   @include e(icon) {
```

Comment:
I don't think using `!important`  is the best option. could we increase the weight of this class, such as

```diff
--- @include e(icon) {
+++ & .#{$namespace}-input__icon {
```
or
```diff
--- @include e(icon) {
+++ .#{$namespace}-icon.#{$namespace}-input__icon {
```

Reply:
OMG, you are right.